hare/eligibility/oracle.go

```diff
@@ -167,15 +185,16 @@ func (o *Oracle) getBeaconValue(ctx context.Context, epochID types.EpochID) (uin
 	}
 
 	value := encodeBeacon(beacon)
-	o.WithContext(ctx).With().Debug("hare eligibility beacon value for epoch",
+	o.WithContext(ctx).With().Info("eligibility: beacon value for epoch",
 		epochID,
 		beacon,
-		log.Uint32("beacon_val", value))
+		log.Uint32("beacon_val", value),
+	)
 	return value, nil
 }
 
 // buildVRFMessage builds the VRF message used as input for the BLS (msg=Beacon##Layer##Round).
-func (o *Oracle) buildVRFMessage(ctx context.Context, layer types.LayerID, round uint32) ([]byte, error) {
+func (o *Oracle) buildVRFMessage(ctx context.Context, nonce types.VRFPostIndex, layer types.LayerID, round uint32) ([]byte, error) {
 	key := buildKey(layer, round)
```

Comment:
need to get rid of the vrfMsgCache. this is what causing the vrf signature to be invalid. once the node sign its own vrf message, it is validating all other nodes' vrf signature with the cached msg

Reply:
I removed the cache and fixed all unit tests that broke because of it; running a new `bors try` now